src/GovUk.Education.ExploreEducationStatistics.Admin/Validators/ValidationErrorMessages.cs

```diff
@@ -112,6 +112,7 @@ public enum ValidationErrorMessages
         RelatedDashboardsSectionContainsEmptyHtmlBlock,
         ReleaseMustContainKeyStatOrNonEmptyHeadlineBlock,
         SummarySectionContainsEmptyHtmlBlock,
+        PublicApiDataSetImportsMustBeCompleted,
```

Comment:
Should we add another checklist item for if the data set version is in the mapping stage still, and not reached 'draft' status yet?

Ultimately I think we only want to allow the approval if all the new data set versions added to this release are in 'draft'

Reply:
Cool 👍. I think it's fine to build in the checklist functionality for `Cancelled` statuses right now, but for the first iteration I don't think we will have any way of cancelling a data set version that is processing so for the time being it won't be possible to reach this status.